### PR TITLE
Further language clean up

### PR DESCRIPTION
**Pull request by antongisli**
edited for readability & conciseness

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1138#" title="Author self-approved">antongisli</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @antongisli. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/api/1-introduction.md [hunk 1]
```diff
@@ -51,7 +52,7 @@ All requests are in the `JSON` format. In general, the request payload has three
 * *Status* contains the status information of the resource. The API does not support creating or modifying the status section. 
 
 <InfoBox>
-Certain update request schema have restricted spec resource definition and certain fields like uid, creation timestamp are not allowed to be modified post creation.
+Certain update request schemas have restricted spec resource definitions and certain fields like uid, creation timestamp are not allowed to be modified post creation.
```
Comment (karl-cardenas-coding) [inline content/api/1-introduction.md:55]:
```suggestion
Certain update request schemas have restricted spec resource definitions, and specific fields like uid and creation timestamp cannot be modified post-creation.
```

File: content/api/1-introduction.md [hunk 2]
```diff
@@ -77,25 +78,25 @@ The API returns standard HTTP response codes:
 | 500 | Operational error. For 500 error code, the server responds with an explicit error code and an error message. |
 
 # Palette API Lifecycle
-The Palette API will remain backward compatible until it's deprecated, and we will keep you informed of its lifecycle phases, which are *Production*, *Sunset*, and *Deprecated*.
+Palette APIs maintain backwards compatibility until deprecation. There are three phases in the lifecycle of the APIs, which are *Production*, *Sunset*, and *Deprecated*. Spectro Cloud will inform users when APIs transition through this lifecycle.
```
Comment (karl-cardenas-coding) [inline content/api/1-introduction.md:81]:
```suggestion
Palette APIs maintain backward compatibility until deprecation. The three API phases in the lifecycle are *Production*, *Sunset*, and *Deprecated*. Spectro Cloud will inform users when APIs transition through this lifecycle.
```

File: content/api/1-introduction.md [hunk 3]
```diff
@@ -77,25 +78,25 @@ The API returns standard HTTP response codes:
...
 
-The API lifecycle also applies to external-facing tools such as Terraform, and they will be managed accordingly.
+The API lifecycle also applies to external-facing tools such as Terraform.
 
 </InfoBox>
 
 # Versioning
 
-The version information is included in the API URI, such as `v1alpha1` or `v1`. Future APIs will increment the version, leaving the earlier version API intact. The existing API request and response schema will be modified to add new attributes or query parameters while maintaining backward compatibility with earlier schema. Prior notice will be given before advancing to the next version, and users will be advised to migrate to the new API.
+The version information is included in the API URI, such as `v1alpha1` or `v1`. Future APIs will increment the version, leaving the earlier version intact. The existing API request and response schema will be modified to add new attributes or query parameters while maintaining backwards compatibility with earlier schemas. Prior notice will be given before advancing to the next version, and users will be advised to migrate to the new API.
```
Comment (karl-cardenas-coding) [inline content/api/1-introduction.md:99]:
```suggestion
The version information is included in the API URI, such as `v1alpha1` or `v1`. Future APIs will increment the version, leaving the earlier version intact. The existing API request and response schema will be modified to add new attributes or query parameters while maintaining backward compatibility with earlier schemas. Prior notice will be given before advancing to the next version, and users will be advised to migrate to the new API.
```

**Review by karl-cardenas-coding — APPROVED** [review]
👍🏻

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1138#" title="Author self-approved">antongisli</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1138#pullrequestreview-1303522393" title="Approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**